.eslintrc.json

```diff
@@ -18,5 +18,5 @@
         "quotes": ["error", "single"],
         "semi": "error"
     },
-    "ignorePatterns": ["**/dist/*"]
+    "ignorePatterns": ["**/dist/*", "src/*.ts"]
```

Comment:
Don't ignore, instead add support for ESLint for TypeScript into the project, using dev deps: @typescript-eslint/eslint-plugin and @typescript-eslint/parser and editing eslintrc.json. More info at https://typescript-eslint.io/

Reply:
I have added support for ts, fixed the errors, but the type any warnings will still be there.
I expect these to be fixed in the future by support from other people as well.
In case we decide to go ahead with the ts version only, maybe more help would join on the repository if usage would increase.